README.md

```diff
@@ -178,14 +178,19 @@ gh migration-audit audit-repo \
 
 #### Installation Authentication
 
-This approach is for use with a GitHub App Installation and takes advantage of [auth-app.js](https://github.com/octokit/auth-app.js). Specifically [use with Octokit](https://github.com/octokit/auth-app.js/?tab=readme-ov-file#usage-with-octokit) implementation is used and creates an instance that is authenticated as an installation, with automated installation token refresh.
+Instead of [token authentication](#token-authentication), installation authentication can be used by providing values for `--app-id`, `--private-key` or `--private-key-file`, and `--app-installation-id`. Use of environment variables is also supported by providing `GITHUB_APP_ID`, `GITHUB_APP_PRIVATE_KEY` or `GITHUB_APP_PRIVATE_KEY_FILE`, and `GITHUB_APP_INSTALLATION_ID`.
```

Comment:
```suggestion
Instead of [token authentication](#token-authentication), installation authentication can be used by providing values for `--app-id`, `--app-installation-id`, and either `--private-key` or `--private-key-file`. Use of environment variables is also supported by providing `GITHUB_APP_ID`, `GITHUB_APP_INSTALLATION_ID`, and either `GITHUB_APP_PRIVATE_KEY` or `GITHUB_APP_PRIVATE_KEY_FILE`.
```